DigitalLearningSolutions.Web/Controllers/LoginController.cs

```diff
@@ -72,6 +74,11 @@ public IActionResult Index(LoginViewModel model)
 
         private void LogIn(AdminUser? adminUser, DelegateUser? delegateUser, bool rememberMe)

```

Comment:
Could you change this method name to "LogInAndRecordAdminSession" to make sure it's not misused?

Reply:
Or alternatively have a recordAdminSession function before "LogIn" is called 🤔 😄